src/app/content/studyGuides/components/Filters.spec.tsx

```diff
@@ -243,7 +243,7 @@ describe('Filters', () => {
 
     expect(dispatch).toHaveBeenCalledWith(
       replace(mockMatch, {
-        search: `colors=yellow&colors=green&colors=blue&colors=purple&modal=${modalUrlName}`,
+        search: `colors=yellow&colors=green&colors=blue&colors=purple&locationIds&modal=${modalUrlName}`,
```

Comment:
Is it ok to have a query value without an `=` here? It's not common to see this. I guess it's a valid thing though.